RoeiJeRot.Logic/Services/MailService.cs

```diff
@@ -91,5 +92,42 @@ public void SendConfirmation(string email, string firstName, DateTime datum, Tim
                 Console.Error.WriteLine("Error while sending mail.");
             }
         }
+
+        public void SendCancelConfirmation(string email, string firstName, DateTime datum)
+        {
+            MailMessage mailtje = new MailMessage();
+            mailtje.From = fromAddress;
+            mailtje.To.Add(new MailAddress(email, firstName));
+            mailtje.Subject = "Bevestiging reservering annuleren";
+            mailtje.Body = $"Beste {firstName}" + Environment.NewLine + Environment.NewLine +
+                           $"Je ontvangt deze mail omdat je een reservering hebt geanuleerd voor een boot op {datum.ToString("d")}." +
+                           Environment.NewLine + Environment.NewLine +
+                           "We hopen u nog een keer weer te zien bij onze vereniging!" + Environment.NewLine + Environment.NewLine +
+                           "Met vriendelijke groeten," + Environment.NewLine +
+                           Environment.NewLine +
+                           "Roeivereniging Roei-je-Rot";
+            try
+            {
+                using (var smtpClient = new SmtpClient("smtp.gmail.com", 587))
+                {
+                    smtpClient.UseDefaultCredentials = false;
+                    smtpClient.Credentials = new NetworkCredential()
```

Comment:
maak een functie om deze credentials uit te lezen en terug te geven. Dit omdat je nu twee keer je wachtwoord en mail in code staat. Als dit veranderd moet iemand dit maar net weten en als die dat niet weet dan worden er geen mails gestuurd.